sda/schemas/isolated/ingestion-completion.json

```diff
@@ -130,17 +130,6 @@
                     }
                 ]
             ],
-            "contains": {
-                "type": "object",
-                "properties": {
-                    "type": {
-                        "const": "sha256"
```

Comment:
So all checksums are optional from now on? since this one is `sha256`

Reply:
that is a mistake